bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/ItemChannelLinkRegistry.java

```diff
@@ -90,31 +87,6 @@ public ItemChannelLink update(ItemChannelLink element) {
         return linkedItems;
     }
 
-    /**
-     * Returns a set of bound things for the given item name.
-     *
-     * @deprecated Will be removed soon. Use {@link ItemThingLinkRegistry#getLinkedThings(String)} instead.
-     *
-     * @param itemName
-     *            item name
-     * @return set of bound things for the given item name
-     */
-    @Deprecated
```

Comment:
Therefore I would leave this method in here and remove the @Deprecated marker, which is wrong.